app/models/holiday.rb

```diff
@@ -18,6 +18,10 @@ def self.merged_for_calendar_view
       .order(:start_at)
   end
 
+  def self.scoped_to_user(user)
+    where(user_id: [user.id, nil])
```

Comment:
OK how about something like `where(user_id: [user.id, BANK_HOLIDAYS = nil])` so it's a little more descriptive?